components/brave_wallet_ui/utils/web3-utils.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
Same here for `2023`

Reply:
Thanks, updated: https://github.com/brave/brave-core/compare/2be3cf85e707fce11b69a56e4e8085018f2e0683..370942a651e22b62667a74533580fa7e706e0dec